chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/UpdateMacraMeasuresJob.java

```diff
@@ -31,6 +34,102 @@
     private static final String OLD_G1G2_SUBSTRING = "ACI";
     private static final String NEW_G1G2_SUBSTRING = "PI";
 
+    private static final String A_1 = "170.315 (a)(1)";
+    private static final String A_2 = "170.315 (a)(2)";
+    private static final String A_3 = "170.315 (a)(3)";
+    private static final String A_10 = "170.315 (a)(10)";
+    private static final String A_13 = "170.315 (a)(13)";
+    private static final String B_1 = "170.315 (b)(1)";
+    private static final String B_2 = "170.315 (b)(2)";
+    private static final String B_3 = "170.315 (b)(3)";
+    private static final String E_1 = "170.315 (e)(1)";
+    private static final String E_2 = "170.315 (e)(2)";
+    private static final String G_8 = "170.315 (g)(8)";
+    private static final String G_9 = "170.315 (g)(9)";
+
+    private static Map<String, List<String>> measuresToRemove;
+    static {
+        measuresToRemove = new HashMap<String, List<String>>();
+        measuresToRemove.put(A_1,
+                Arrays.asList("EP Stage 2",
```

Comment:
There are plenty of repeats of Strings within these Arrays which could be externalized as they leave room for error as is